bitrise.yml

```
@@ -1,57 +1,69 @@
 ---
-format_version: "11"
+format_version: '11'
 default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 project_type: react-native
 
 trigger_map:
 - push_branch: "*"
-  workflow: primary
+  pipeline: primary
 - pull_request_source_branch: "*"
-  workflow: primary
-
-workflows:
+  pipeline: primary
 
+pipelines:
   primary:
     description: Runs JavaScript tests, and verifies iOS and Android builds
-    before_run:
-    - _setup
-    after_run:
-    - _test-js
-    - _build-ios
-    - _build-android
+    stages:
+    - test-and-build-stage: {}
+
+stages:
+  test-and-build-stage:
+    abort_on_fail: true
+    workflows:
+    - test-js: {}
+    - build-ios: {}
+    - build-android: {}
 
+workflows:
   _setup:
     steps:
-    - git-clone@6: { }
-    - restore-npm-cache@1: { }
+    - git-clone@6: {}
```

```suggestion
    - git-clone@7: {}
```

For maximum performance 😎